packages/integration_test/lib/integration_test.dart

```diff
@@ -167,11 +167,33 @@ https://flutter.dev/docs/testing/integration-tests#testing-on-firebase-test-lab
   /// side.
   final CallbackManager callbackManager = driver_actions.callbackManager;
 
-  /// Taking a screenshot.
+  /// Takes a screenshot.
   ///
-  /// Called by test methods. Implementation differs for each platform.
-  Future<void> takeScreenshot(String screenshotName) async {
-    await callbackManager.takeScreenshot(screenshotName);
+  /// On Android, you need to call `convertFlutterSurfaceToImage()`, and set the
+  /// `framePolicy` to `LiveTestWidgetsFlutterBindingFramePolicy.fullyLive`
+  /// prior to taking a screenshot.
```

Comment:
Is this not the case on web?

We should probably explain why this is the case here.

Reply:
No need for "Set `framePolicy` to `LiveTestWidgetsFlutterBindingFramePolicy.fullyLive". I changed it, so only pumpAndSettle is necessary.